src/components/Track/track.css

```diff
@@ -0,0 +1,31 @@
+.track {
+  display: grid;
+  align-items: center;
+  grid-template: auto / auto 1fr auto;
```

Comment:
Hey, good job! Just one thing: grid-template goes both directions, vertical and horizontal. You only have three object in one column, so you could use grid-template-columns: auto 1fr auto;.

Reply:
i'll have a check on that 👍🏻  thank you Bela